pkg/util/apiserver/cors_test.go

```diff
@@ -0,0 +1,141 @@
+// Copyright (C) 2020, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package apiserver
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/verrazzano/verrazzano-operator/pkg/api/instance"
+)
+
+var allowedHTTPMethods = []string{"POST", "GET", "OPTIONS", "PUT", "DELETE"}
+var allowedHeaders = []string{"Accept", "Content-Type", "Content-Length", "Accept-Encoding", "X-CSRF-Token", "Authorization"}
+
+//TestEnableCors tests that the EnableCors method adds the expected Access-Control-Allow-Origin header to the response
+func TestEnableCors(t *testing.T) {
+	type args struct {
+		verrazzanoURI string
+		writer        http.ResponseWriter
+	}
+	tests := []struct {
+		name                string
+		origin              string
+		expectedAllowOrigin string
+		args                args
+	}{
+		{"With Verrazzano URI set", "https://console.somesuffix.xip.io",
+			"https://console.somesuffix.xip.io",
+			args{verrazzanoURI: "somesuffix.xip.io", writer: http.ResponseWriter(httptest.NewRecorder())}},
+		{"Without Verrazzano URI set", "http://localhost",
+			"", args{verrazzanoURI: "", writer: http.ResponseWriter(httptest.NewRecorder())}},
+		{"Origin does not match", "http://localhost",
+			"", args{verrazzanoURI: "somesuffix.xip.io", writer: http.ResponseWriter(httptest.NewRecorder())}},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			//GIVEN the Verrazzano URI suffix is set
+			instance.SetVerrazzanoURI(tt.args.verrazzanoURI)
+			req, err := http.NewRequest("GET", "http://someUrl", nil)
+			assert.Nil(t, err, "Error creating test request")
+			req.Header.Add("origin", tt.origin)
+
+			//WHEN EnableCors is called
+			EnableCors(req, &tt.args.writer)
+
+			//THEN it returns the expected Access-Control-Allow-Origin header value
+			assert.Equal(t, tt.expectedAllowOrigin, tt.args.writer.Header().Get("Access-Control-Allow-Origin"))
+		})
+	}
+}
+
+//TestSetupOptionsResponse tests that the SetupOptionsResponse method adds the expected CORS related headers to the response
+func TestSetupOptionsResponse(t *testing.T) {
+	type args struct {
+		verrazzanoURI string
+		writer        http.ResponseWriter
+	}
+	tests := []struct {
+		name                 string
+		origin               string
+		expectedAllowOrigin  string
+		expectedAllowMethods []string
+		expectedAllowHeaders []string
+		args                 args
+	}{
+		{"With Verrazzano URI set", "https://console.somesuffix.xip.io",
```

Comment:
nitpick, put each item on a new line so it is easier to read and maintain

Reply:
done - also added the field names to improve readability